app/assets/stylesheets/layout/_work_package.sass

```diff
@@ -105,6 +105,14 @@
   width:      100%
   +allow-vertical-scrolling
   padding:    0 25px 0 25px
+  &.-create-mode
+    top: 0
+    h2
+      margin-top: 0.375em
+    .single-attribute.subject
```

Comment:
See my comment on the corresponding container below. I'd say one could replace this with the follwoing selector:

````CSS
.work-package-field.work-packages--details--subject
````